CHANGELOG.md

```diff
@@ -5,13 +5,17 @@ r# NEXT RELEASE
 * Cut the runtime of aggregate operations on large dictionaries in half ([PR #5864](https://github.com/realm/realm-core/pull/5864)).
 * Improve performance of aggregate operations on collections of objects by 2x to 10x ([PR #5864](https://github.com/realm/realm-core/pull/5864)).
 * Adding support in the C API for `realm_results_find` and supporting all native types for `realm_results_get()` ([PR 5875](https://github.com/realm/realm-core/pull/5875)).
+* Reconstruct app url after a migration ([#5648](https://github.com/realm/realm-core/issues/5648)).
```

Comment:
Did this get merged/rebased incorrectly with your url update PR?

Reply:
Yes - it was a regular pull without any conflicts and added these changes to Release 12.7.0...